packages/ripple-tide-grant/index.js

```diff
@@ -0,0 +1,89 @@
+import mime from 'mime-types'
+import { getField, getLinkFromField } from '@dpc-sdp/ripple-tide-api/src/services/utils'
+import components from './component-loader'
+
+// Utils
+const formatAudiences = (audiences) => {
```

Comment:
This one seems very specific to parsing a single word out of a giant object, I don't think it needs to go out to utils